compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/scene/skia/SwingSkiaLayerComponent.desktop.kt

```diff
@@ -66,7 +66,7 @@ internal class SwingSkiaLayerComponent(
                 mediator.preferredSize
             }
 
-            override fun getAccessibleContext(): AccessibleContext? {
+            override fun getAccessibleContext(): AccessibleContext {
```

Comment:
Please revert unrelated changes

Reply:
It's a compilation error without this one on my machine